### PR TITLE
Feature/price inputs

### PR DESCRIPTION
**Pull request by marin250189**
Add prices related input when they are hidden

### CODE REVIEW

File: assets/js/bf_woo_builder.js [hunk 1]
```diff
@@ -174,18 +182,47 @@ jQuery(document).ready(function ($) {
...
+                sales_price_amount.show();
+            }
+            else{
+                sales_price_amount_row.hide();
             }
         });
 
         $('select[name="buddyforms_options[form_fields][' + field_id + '][product_sales_price_dates]"]').change(function () {
-            var sale_prices = $('select[name="buddyforms_options[form_fields][' + field_id + '][product_sales_price]"]').val();
+            /*var sale_prices = $('select[name="buddyforms_options[form_fields][' + field_id + '][product_sales_price]"]').val();
```
Comment (gfirem) [inline assets/js/bf_woo_builder.js]: @victorveas15 please remove comment code

File: assets/js/bf_woo_builder.js [hunk 2]
```diff
@@ -174,18 +182,47 @@ jQuery(document).ready(function ($) {
...
+            }
+            else{
+                regular_price_amount_row.hide();
+            }
+        });
+
         $('select[name="buddyforms_options[form_fields][' + field_id + '][product_sales_price]"]').change(function () {
             if ($(this).val() === 'hidden') {
-                $('select[name="buddyforms_options[form_fields][' + field_id + '][product_sales_price_dates]"]').val('hidden');
+                //$('select[name="buddyforms_options[form_fields][' + field_id + '][product_sales_price_dates]"]').val('hidden').change();
```
Comment (gfirem) [inline assets/js/bf_woo_builder.js]: @victorveas15 please remove commented code

File: assets/js/bf_woo_builder.js [hunk 3]
```diff
@@ -174,18 +182,47 @@ jQuery(document).ready(function ($) {
             }
         });
 
+        $('select[name="buddyforms_options[form_fields][' + field_id + '][product_regular_price]"]').change(function () {
+            if ($(this).val() === 'hidden') {
+
```
Comment (gfirem) [inline assets/js/bf_woo_builder.js:187]: @victorveas15 please remove unnecessary empty lines

File: assets/js/bf_woo_general_settings.js [hunk 4]
```diff
@@ -221,6 +221,7 @@ jQuery(document).ready(function ($) {
             var regularPrice = general_settings_param.product_regular_price;
             if (regularPrice === "hidden") {
                 hide_general_regular_price = true;
+                $('#_regular_price').val(general_settings_param.regular_price_amount);
```
Comment (gfirem) [inline assets/js/bf_woo_general_settings.js]: @victorveas15 please add a validation for the next vars, to js errors.
- regular_price_amount
- sales_price_amount
- product_sales_start_date
- product_sales_end_date

**Review by gfirem — CHANGES_REQUESTED** [review]

File: assets/js/bf_woo_general_settings.js (hunk 4 above)
Comment (marin250189) [reply assets/js/bf_woo_general_settings.js]: done

**Review by gfirem — APPROVED** [review]

**Pull request merged**